website/docs/r/athena_named_query.html.markdown

```diff
@@ -17,15 +17,34 @@ resource "aws_s3_bucket" "hoge" {
   bucket = "tf-test"
 }
 
+resource "aws_kms_key" "test" {
+  deletion_window_in_days = 7
+  description             = "Terraform Acceptance Testing"
```

Comment:
Nit: Since this is example documentation and not acceptance testing, this should likely be replaced here, e.g.

```suggestion
  description             = "Athena KMS Key"
```